Framework/src/SetUpTopTagger.cc

```diff
@@ -12,12 +12,12 @@ SetUpTopTagger::SetUpTopTagger(NTupleReader& tr,
     AK4Inputs_                 (nullptr),
     AK8Inputs_                 (nullptr),
     ak4Filter_                 (nullptr),
-    Jets_                      (tr.getVec<TLorentzVector>("Jets")),
+    Jets_                      (tr.getVec<TLorentzVector>("Jets"+myVarSuffix_)),
     Jets_bJetTagDeepCSVtotb_   (tr.getVec<double>("Jets_bJetTagDeepCSVtotb")),
```

Comment:
These other jet variables should also have suffixes.

Reply:
In fact all jet variables should have a suffix.

Reply:
Ahh, I suppose it would help if I put the suffix right after the `Jets_` rather than simply at the end and it will work.

Reply:
@cmadrid1 Hmm, so I still find that the ones I neglected do not have a variation version...

Reply:
Here is the list of AK4 options and there is a similarr list for AK8 jets. https://github.com/StealthStop/Framework/blob/master/Framework/include/PrepNTupleVars.h#L130-L151
If you don't see the jet variable that we want there then please add to this list.

Reply:
> 
> 
> Here is the list of AK4 options and there is a similarr list for AK8 jets. https://github.com/StealthStop/Framework/blob/master/Framework/include/PrepNTupleVars.h#L130-L151
> If you don't see the jet variable that we want there then please add to this list.

Excellent, thanks for this !

Reply:
Got all variation versions of these after some changes in `PrepNTuplesVars.h`